packages/blockchain-wallet-v4-frontend/src/modals/BuySell/CryptoSelection/CryptoSelector/index.tsx

```diff
@@ -134,7 +134,7 @@ class CryptoSelector extends React.Component<InjectedFormProps<{}, Props> & Prop
       fiatCurrency: this.props.walletCurrency,
       orderType: this.state.orderType,
       pair,
-      step: 'ENTER_AMOUNT',
+      step: 'SELL_ENTER_AMOUNT',
```

Comment:
Should be using `getEnterAmountStepType` here?

Reply:
This is a part of `handleSell` method. We also have `handleBuy` method in this component. I assume `handleSell` will never be used for buy 🤞